.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "sherlodoc"]
+	path = sherlodoc
+	url = https://github.com/Julow/sherlodoc
+	branch = odoc3_compat
```

Comment:
These changes

Reply:
Oh yes sorry, I accidentally pushed them when pushing the last commit.

Reply:
This PR branch now only contains the relevant commits. If you want to build the `odoc` doc with `dune exec -- odoc_driver -p odoc`, then use the [`new_driving_guide_work`](https://github.com/panglesd/odoc/tree/new_driving_guide_work) branch.